### ISSUE TITLE
Transcription at varying levels of granularity

### ISSUE THREAD

**Issue by mikeapp** [IIIF/iiif-stories#13]
Many of the medieval use cases (T-PEN, DM, etc.) address line-by-line transcription.  In other cases the use may wish to transcribe at the paragraph or even page level.   At Yale, we have cases where scholars will be transcribing printed editions that contain minor textual variations.  The transcription will likely take place at the page level, with an existing electronic version of the text being pasted into the transcription annotation and then modified.

**Comment by blalbrit** [comment]
Agreed with above - and this is probably the norm for translation use-cases.

**Comment by sdellis** [comment]
I could imagine an interface which interprets hard carriage returns as a new line.  I could also imagine the transcription window having "line numbers" similar to how GitHub does this for code.  I like T-PEN's attempt to identify all the lines automatically, but I can also imagine other ways to quickly associate lines with canvas regions.

**Comment by aeschylus** [comment]
Have a look at issue #14. How can we represent paragraph=>line=>word nesting to drive a precise display?
